backend/Pipfile

```diff
@@ -18,7 +18,6 @@ django-debug-toolbar = "*"
 
 [packages]
 django = "*"
-psycopg2 = "*"
```

Comment:
Just an FYI taht this change shouldn't be committed -- pyscopg2 is necessary for production deployments

Reply:
Yeah for sure; I can't run the server when that is included (can't get it to work on mac with pycharm) so I was planning to remove that for development and then add it back before merging.  Is there a better way to do it?